src/twisted/internet/defer.py

```diff
@@ -1735,12 +1737,16 @@ def _inlineCallbacks(
                     lineno,
                 )
 
-            status.deferred.callback(e.value)
-            return
+            should_callback = True
```

Comment:
I think this is cleaner with `break`


```python
while 1:
    ...
    callbackVal = e.value
    break
    ...
    callbackVal = e.value
    break

status.deferred.callback(callbackVal)
```

Reply:
here's the change https://github.com/twisted/twisted/commit/39ee52c3db975308a0d0ba336d4adb2673d48c5d